Formula/u/ucommon.rb

```diff
@@ -1,16 +1,16 @@
 class Ucommon < Formula
   desc "GNU C++ runtime library for threads, sockets, and parsing"
   homepage "https://www.gnu.org/software/commoncpp/"
-  url "https://ftp.gnu.org/gnu/commonc++/ucommon-7.0.0.tar.gz"
-  sha256 "6ac9f76c2af010f97e916e4bae1cece341dc64ca28e3881ff4ddc3bc334060d7"
+  url "https://ftp.debian.org/debian/pool/main/u/ucommon/ucommon_7.0.1.orig.tar.gz"
```

Comment:
No tarball at gnu.org?

Reply:
yeah, a bit weird, but this is the 7.0.1 release thread, https://lists.gnu.org/archive/html/bug-commoncpp/2020-09/msg00001.html

Reply:
I can leave a note there and ask why the gnu tarball missing.

Reply:
Sounds good!

Reply:
looks like no reply yet, https://lists.gnu.org/archive/html/bug-commoncpp/2024-08/msg00000.html :(